framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -372,7 +375,7 @@ public static String parseMethod(String methodSign, String input) {
     byte[] selector = new byte[4];
     System.arraycopy(Hash.sha3(methodSign.getBytes()), 0, selector, 0, 4);
     //System.out.println(methodSign + ":" + Hex.toHexString(selector));
-    if (StringUtils.isNullOrEmpty(input)) {
+    if (StringUtils.isEmpty(input)) {
```

Comment:
why you use isEmpty instead of isNullOrEmpty

Reply:
they provide the same implementation, isNullOrEmpty  can be replaced with isEmpty, also, no need to import org.pf4j.util.StringUtils